src/tests/pages/ActorPage.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { useAppSelector } from '../../store/store';
+import ActorPage from '../../pages/ActorPage/ActorPage';
+import renderWithProviders from '../../mockedData/test-utils';
+import dataActor from '../../mockedData/data-actor.json';
+import dataCards from '../../mockedData/data-movies.json';
+import dataImages from '../../mockedData/data-images.json';
+
+jest.mock('../../store/store', () => ({
+  __esModule: true,
+  ...jest.requireActual('../../store/store'),
+  useAppSelector: jest.fn(),
+}));
+
```

Comment:
To make your work simpler and the test file more readable, you can create a function that will return a configurable mock object. 

For example:  const getUseAppSelectorMock = ({lang = 'en', genresArray = [{ id: 0, name: 'super action' }], loading = false, data = dataActor, images = null, films = dataCards, error = ''}) => ({lang, genresArray, loading, data, images, films, error});

So, that will provide you a simple way to change any of params what you need.